lib/buildpackutil.py

```diff
@@ -479,7 +479,7 @@ def bypass_loggregator_logging():
     # Necessary since we store these in cloud portal as strings.
     try:
         bypass_loggregator = strtobool(env_var)
-    except ValueError as e:
+    except ValueError:
```

Comment:
The commit messages says that we are just updating libraries. But this is non-library code. If we are fixing linting issues, can we do this in a separate commit, please?

Reply:
no longer in mr.